packages/lesswrong/components/localGroups/LocalGroupsList.tsx

```diff
@@ -43,16 +40,9 @@ const LocalGroupsList = ({terms, children, classes, showNoResults=true, showLoad
         {results && results.map((group) => <LocalGroupsItem key={group._id} group={group} />)}
       </div>
       <SectionFooter>
-        {(showLoadMore) &&
-          <div className={classes.loadMore}>
-            <LoadMore
-              loadMore={loadMore}
-              count={count}
-              totalCount={totalCount}
-            />
-            { !dimWhenLoading && showLoading && loadingMore && <Loading />}
-          </div>
-        }
+        <div className={classes.loadMore}>
+          <LoadMore {...loadMoreProps} />
+        </div>
```

Comment:
The changes in this file are just to make sure we stop showing the "Load More" button once we've loaded the whole list

Reply:
Good diff.